setup.cfg

```diff
@@ -20,7 +20,7 @@ universal = 1
 add_ignore = D401
 
 [pycodestyle]
-exclude = docs/conf.py
+exclude = docs/conf.py, tests/migrator/*
```

Comment:
I had to do this, there is a lot of compared string types which are not getting handled by black